src/rules/linebreakStyleRule.ts

```diff
@@ -0,0 +1,61 @@
+import * as ts from "typescript";
```

Comment:
Let's just remove the `CR` case from this rule all together for clarity. Shouldn't be too hard to add in later if need be (but that's unlikely anyways)